k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml

```diff
@@ -117,6 +117,7 @@
   dmap:
     "sha256:c1bedac8758029948afe060bf8f6ee63ea489b5e08d29745f44fab68ee0d46ca": [ "v2.2.2" ]
     "sha256:2de1d15fc1f2a33618c61ff8d499ef22a7b2b8961952f059656a245b98498ba4": [ "v3.0.0" ]
+    "sha256:b9d198c84270f8fe984c32eba128ec7589f9de791059585c920517122667a74d": [ "v3.0.1" ]
```

Comment:
One more question, where does this image come from and how did it get built? I don't see the v3.0.1 tag in the repo:
https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner

cc @kmova

Reply:
As a result of https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/pull/89 being merged, it was [built in prow](https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/post-nfs-ganesha-server-and-external-provisioner-push-images/1475150741722107904) and then [pushed to staging](https://console.cloud.google.com/gcr/images/k8s-staging-sig-storage/global/nfs-provisioner@sha256:b9d198c84270f8fe984c32eba128ec7589f9de791059585c920517122667a74d/details).

As for the tag, @kvaps and team would have to handle that, but it could be applied after the fact.  @kvaps?

Reply:
The process I've seen is for the prow build job to build and push the image based on the git tag. So the version is released in the github repo first, then the staging image gets promoted.

Reply:
@kvaps do you have any objections to tagging the current state of nfs-provisioner as v3.0.1 per the above?

Reply:
Hi, I'm sorry I little lost sight of that.
I have no objections. Is there anything I should do for that from my side?

Reply:
For starters, if you could create a `v3.0.1` tag in https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner, we can see if that triggers an image tag in gcr.io via prow.

Reply:
Not sure if I have proper permissions for that

remote: Permission to kubernetes-sigs/nfs-ganesha-server-and-external-provisioner.git denied to kvaps.
fatal: unable to access 'https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/': The requested URL returned error: 403

Reply:
@kmova are you able to help do a release?

Reply:
/hold

till we release a new version for the above repo,

Reply:
/cc @jsafrane

Reply:
Sorry, been out of business for a few months now. Yes @msau42 I will pick up the release task.

Reply:
3.0.1 image is now available.

Reply:
Thanks @kmova.. @yselkowitz can you please update the sha from the recent image built?

Reply:
Thank you @kmova.  Updated the PR with the new shasum.
/hold cancel